### PR TITLE
Frontend UI

### PR DESCRIPTION
**Pull request by SevenDaysDA**

### CODE REVIEW

**Comment by timbmg** [comment]
Thanks for the PR @SevenDaysDA!

I think having a mixed nav with items not having, and having items looks a little weird. @Rachneet suggested to put the Feedback and Github items all the way to the right (i.e. left of the sign in / username) and separate it with a `|` from the sing-in / username button. What do you think?

File: square-frontend/src/components/Footer.vue [hunk 1]
```diff
@@ -7,6 +7,21 @@
...
+                  </svg>
+                </a>
+          </ul>
+        </li>
+      </ul>
+    </div>
+    <div class="container">
+      <ul class="nav justify-content-center">
+        <li class="nav-item"><img :src="`${publicPath}ukp_logo.png`" alt="Ubiquitous Knowledge Processing" width="110" height="40" class="px-2"></li> 
+        <li class="nav-item"><img :src="`${publicPath}tud_logo.png`" alt="TU Darmstadt" width="120" height="40" class="px-2"></li>
```
[the remark this answers is not in the document]
Comment (timbmg) [reply square-frontend/src/components/Footer.vue:24]: also i would increase the height; ~60 looked good to me. I think ideally the font size of the TU-DA logo matches the font-size of the rest of the footer.

**Review by timbmg — CHANGES_REQUESTED** [review]

**Comment by SevenDaysDA** [comment]
Hi @timbmg ,I agree.
The mixed elements look a bit off.
I moved them to the right.

File: square-frontend/src/components/Footer.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (SevenDaysDA) [reply square-frontend/src/components/Footer.vue:24]: Done. 
Thank you for the review.

**Pull request merged**